PWGGA/GammaConv/macros/AddTask_GammaConvV1_pp.C

```diff
@@ -712,6 +712,19 @@ void AddTask_GammaConvV1_pp(
     cuts.AddCutPCM("00010113", "0020008929700000iih0400000", "0152103500000000"); // pt dep Qt, chi2-psipair exp
     cuts.AddCutPCM("00010113", "0020008929700000i280400000", "0152103500000000"); // pt dep Qt, open chi2-psipair
     cuts.AddCutPCM("00010113", "00200089297000001ih0400000", "0152103500000000"); // chi2-psipair, open Qt
+  } else if (trainConfig == 304) { // dEdx recalib
```

Comment:
A configuration like this can not be approved. Please split it in MB, TRD triggers, Phos triggers, EMC triggers, and others!
Also it might be recommended to directly add the calo cutnumbers for the trigger rejection.